contracts/RiskManagerV1.sol

```diff
@@ -454,7 +455,7 @@ contract RiskManagerV1 is IRiskManager, Auctioneer, Ownable {
         IDeposit deposit = IDeposit(auctionToDeposit[address(auction)]);
         // Make sure the deposit was not liquidated outside of Coverage Pool
         require(
-            deposit.currentState() == DEPOSIT_LIQUIDATION_IN_PROGRESS_STATE,
+            isDepositLiquidationInProgress(deposit),
```

Comment:
After the change from #84, we need to add this check to `onAuctionFullyFilled`.

Reply:
Sure. Here it is: https://github.com/keep-network/coverage-pools/pull/109/commits/9478d91c786edfa234ef548783220003648c8097